ui/components/app/assets/auto-detect-nft/auto-detect-nft-modal.tsx

```diff
@@ -117,15 +114,17 @@ function AutoDetectNftModal({ isOpen, onClose }: AutoDetectNftModalProps) {
         </ModalBody>
         <ModalFooter
           onSubmit={() => handleNftAutoDetection(true)}
-          submitButtonProps={{
-            children: t('allow'),
-            block: true,
-          }}
           onCancel={() => handleNftAutoDetection(false)}
           cancelButtonProps={{
             children: t('notRightNow'),
             block: true,
+            variant: ButtonVariant.Link,
+          }}
+          submitButtonProps={{
+            children: t('allow'),
+            block: true,
           }}
+          containerProps={{ flexDirection: FlexDirection.ColumnReverse }}
```

Comment:
Added these updates @sahar-fehri 

<img width="751" alt="Screenshot 2024-07-22 at 10 02 39 AM" src="https://github.com/user-attachments/assets/27503d7b-061c-49ad-b462-487e21725e8d">

Reply:
Thanks a lot @georgewrmarshall ! The  `flexDirection: FlexDirection.ColumnReverse` is way better 🙏